kaleidoscope_collaborative/lib/screens/AddRating/review_page_1_1_overallRatingPage.dart

```diff
@@ -72,76 +81,75 @@ class _AddReviewPageState extends State<AddReviewPage> {
                 ),
               ],
             ),
-            Column( 
+            Column(
               // crossAxisAlignment: CrossAxisAlignment.stretch,
               mainAxisAlignment: MainAxisAlignment.center,
               children: <Widget>[
                 SizedBox(height: 20),
-            Text(
-              'How would you rate this business?',
-              textAlign: TextAlign.center,
-              style: TextStyle(
-                fontSize: 20,
-                fontWeight: FontWeight.bold,
-              ),
-            ),
-            SizedBox(height: 48),
-            
-            Row(
-              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-              children: 
-              List.generate(5, (index) {
-                
-                return IconButton(
-                  icon: Stack(
-                    alignment: Alignment.center,
-                    children: <Widget>[
-                      // Circle icon as the background
-                      Icon(
-
-                        index < overallRating ? Icons.circle : Icons.circle,
-                        color: index < overallRating ? Color(0xFF6750A4) : Colors.grey,
-                        size: 60, 
-                      ),
-                      // Star icon on top of the circle
-                      Align(
+                Text(
+                  'How would you rate this business?',
+                  textAlign: TextAlign.center,
+                  style: TextStyle(
+                    fontSize: 20,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+                SizedBox(height: 48),
```

Comment:
Please replace the magic number with dynamic measurement to avoid the pixel overflow issue.